### PR TITLE
New script to toggle delayed descriptions for characters.

### PR DESCRIPTION
**Pull request by CyrilleB79**
### Link to issue number:
https://nvda.groups.io/g/nvda/message/100197

### Summary of the issue:

A script to toggle delayed character description has been asked.

### Description of user facing changes

Added a script to toggle delayed character description.

Keep this script unassigned because it does not seem to me that this feature deserve a predefined gesture.

### Description of development approach

Added the script in global commands, as other such scripts.

### Additional possible changes
I am always a bit disturbed by the wording "Delayed character descriptions" or "Delayed descriptions for characters" with the plural form, specifically in my language (French). The other options, on the contrary, are worded with singular form, e.g. we do not have "Font sizes".
If you agree, while at it, I may reword this option using singular form.
What do you think? CC @Qchristensen 

If it's not an issue in English, I'll see on French translation side only.

### Testing strategy:
Manual tests:
I have assigned the script and used it. I have checked that:
- the script correctly reports its action
- the option in the GUI is changed adequately
- the feature is enabled/disabled according to the action of the script
- input help is working well

### Known issues with pull request:
None.

### Change log entries:
`An unassigned command has been added to toggle delayed character descriptions.`

### Code Review Checklist:

- [x] Pull Request description:
  - description is up to date
  - change log entries
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] API is compatible with existing add-ons.
- [x] Documentation:
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] Security precautions taken.

### CODE REVIEW

**Comment by CyrilleB79** [comment]
Failing test is "lien    Robot.symbolPronunciationTests.tableHeaderSymbols".
Seems completely unrelated.

**Comment by seanbudd** [comment]
>I am always a bit disturbed by the wording "Delayed character descriptions" or "Delayed descriptions for characters" with the plural form, specifically in my language (French). The other options, on the contrary, are worded with singular form, e.g. we do not have "Font sizes".

The current form reads well to me, what do you propose the alternative be?

**Review by seanbudd — APPROVED** [review]

**Comment by CyrilleB79** [comment]
> > I am always a bit disturbed by the wording "Delayed character descriptions" or "Delayed descriptions for characters" with the plural form, specifically in my language (French). The other options, on the contrary, are worded with singular form, e.g. we do not have "Font sizes".
> 
> The current form reads well to me, what do you propose the alternative be?

Here is what I have in mind:
* Option name: "Delayed descriptions for characters on cursor movement" to be replaced by "Delayed description for characters on cursor movement"
* In this PR's code: "delayed character descriptions on/off" to be replaced by "delayed character description on/off"
* In this PR's change log entry: "An unassigned command has been added to toggle delayed character descriptions." to be replaced by "An unassigned command has been added to toggle delayed character description."
* Older change log will not be updated as we usually do not modify change log items of a previous release.

Again, English is not my native language. So just let me know if English formulation is correct. Maybe we are just in those cases where English uses plural and French singular. E.g. in the following example:
"The students take off their coats and take out their notebooks.", French would translate "coats" and "notebooks" with singular form since there is only one coat and one notebook per student.

Note: I realize that I have put singular form for "script_toggleDelayedCharacterDescription". If we keep the plural form in English, I should update the script's name.

**Comment by seanbudd** [comment]
@CyrilleB79 - I think the current form reads better in English

File: source/globalCommands.py [hunk 1]
```diff
@@ -943,6 +943,22 @@ def script_cycleSpeechSymbolLevel(self,gesture):
 		# %s will be replaced with the symbol level; e.g. none, some, most and all.
 		ui.message(_("Symbol level %s") % name)
 
+	@script(
+		# Translators: Input help mode message for toggle delayed character description command.
+		description=_("Toggles on and off delayed descriptions for characters on cursor movement"),
+		category=SCRCAT_SPEECH,
+	)
+	def script_toggleDelayedCharacterDescription(self, gesture: inputCore.InputGesture) -> None:
```
Comment (seanbudd) [inline source/globalCommands.py:951]:
```suggestion
	def script_toggleDelayedCharacterDescriptions(self, gesture: inputCore.InputGesture) -> None:
```

**Pull request merged**